### ISSUE TITLE
Media Blitz becomes targeted agenda

### ISSUE THREAD

**Issue by Dirjel** [mtgred/netrunner#4933]
If you play Media Blitz and target Jumon, then drag Media Blitz out of the current area and into your grip/Archives, it BECOMES the agenda it was targeting:
![image](https://user-images.githubusercontent.com/42731111/72307824-deaa6980-3638-11ea-986e-32053292db14.png)

As you see, I have 5 copies of Jumon in play.  2 of them, including one of the ones the Runner "scored" is actually a copy of Media Blitz (the Jumon in HQ was also a Media Blitz I played for further testing).

**Comment by NoahTheDuke** [comment]
Yeah, the only way we can make Media Blitz work is by changing the name on the card lol. Until we get a fix up, you can use  `/trash` to move it to the discard, I think.
